### PR TITLE
MTM-60356 - changelog for a bugfix regarding capability to create measurement series without value

### PR DESCRIPTION
**Pull request by mbrz-c8y**

### CODE REVIEW

File: content/change-logs/platform-services/cumulocity-10-20-566-0-fixed-measurement-series-without-value.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+date: 2024-09-23
+title: Fixed the capability to create measurement series without value
```
Comment (MWindel) [inline content/change-logs/platform-services/cumulocity-10-20-566-0-fixed-measurement-series-without-value.md]:
```suggestion
title: Measurement series can now be created without value
```

**Review by schmiel-cumulocity — APPROVED** [review]

File: content/change-logs/platform-services/cumulocity-10-20-566-0-fixed-measurement-series-without-value.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  - value: component-OG_650_b2
+    label: Core platform
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-60356
+version: 10.20.566.0
+---
+
+Fixed a regression in time series measurements that allowed to create a measurement series without value.
```
Comment (MWindel) [inline content/change-logs/platform-services/cumulocity-10-20-566-0-fixed-measurement-series-without-value.md]: Please add previous status.

File: content/change-logs/platform-services/cumulocity-10-20-566-0-fixed-measurement-series-without-value.md (hunk 1 above)
Comment (mbrz-c8y) [reply content/change-logs/platform-services/cumulocity-10-20-566-0-fixed-measurement-series-without-value.md]: I changed wording a little bit. Now such measurements can no longer be created.

File: content/change-logs/platform-services/cumulocity-10-20-566-0-fixed-measurement-series-without-value.md (hunk 2 above)
Comment (mbrz-c8y) [reply content/change-logs/platform-services/cumulocity-10-20-566-0-fixed-measurement-series-without-value.md]: What do you exactly mean by 'previous status'? How it used to work before the change?

Previously, it was possible to create measurement without value which was discrepant with the [API documentation](https://www.cumulocity.com/api/core/#operation/postMeasurementCollectionResource) (`The value is mandatory for a fragment.`)

**Review by jaro-herod — APPROVED** [review]

File: content/change-logs/platform-services/cumulocity-10-20-566-0-fixed-measurement-series-without-value.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+date: 2024-09-23
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-566-0-fixed-measurement-series-without-value.md]:
```suggestion
date: 
```

**Review by karol-c8y — APPROVED** [review]

File: content/change-logs/platform-services/cumulocity-10-20-566-0-fixed-measurement-series-without-value.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  - value: component-OG_650_b2
+    label: Core platform
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-60356
+version: 10.20.566.0
+---
+
+Fixed a regression in time series measurements that allowed to create a measurement series without value.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-566-0-fixed-measurement-series-without-value.md]:
```suggestion
Previously, it was possible to create a measurement series without value. This issue has been fixed as the value is mandatory. Now a value must be provided on measurement creation. 
```

File: content/change-logs/platform-services/cumulocity-10-20-566-0-fixed-measurement-series-without-value.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+---
+date: 
+title: Measurement series without value can no longer be created
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-566-0-fixed-measurement-series-without-value.md]:
```suggestion
title: Measurement series can no longer be created without value
```

**Comment by BeateRixen** [comment]
@mbrz-c8y Please check my suggestions and approve

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**